[WIP] Pointshop Update

PYROTEKNIK opened the pull request:
i made an attempt to improve the overall design
![image](https://user-images.githubusercontent.com/3793593/123677916-85b59d80-d7fa-11eb-8da1-b7dd415ee01b.png)

swampservers committed:
overwrite2

swampservers committed:
merge

swampservers committed:
pee print

swampservers commented on gamemodes/cinema/gamemode/modules/scoreboard/cl_playerlist.lua:
```diff
@@ -153,48 +216,116 @@ end
+    end)
 end
 
+GetCountryData()
```
Can you not do this extraneous http request until someone has actually moused over the thing or whatever

PYROTEKNIK replied:
i don't think there's an event for when tooltips appear

PYROTEKNIK committed:
Update sh_init.lua

removed sample inventory code

PYROTEKNIK committed:
oops!

PYROTEKNIK committed:
fixed file includes

PYROTEKNIK committed:
Update menu.lua

removed debug function

PYROTEKNIK committed:
removed unused element

PYROTEKNIK committed:
fixed grey bars

swampservers commented on lua/autorun/client/stylesheet.lua:
```diff
@@ -112,4 +288,159 @@ function draw.GradientShadowUp(x, y, w, h, alpha)
\ No newline at end of file
+end
+
+surface.CreateFont('SS_Heading', {
```
Can you put swampshop-specific styles in a different file? lua/swampshop/stylesheet.lua (and make sure to include it)

PYROTEKNIK replied:
i kinda figured most of this would probably be useful to reuse elsewhere

PYROTEKNIK committed:
removed nsfw flagging

PYROTEKNIK committed:
Update customizer.lua

style consistency

PYROTEKNIK committed:
Update menu.lua

removed debug

PYROTEKNIK committed:
Update stylesheet.lua

dark stripes have a higher saturation value, kinda pops a bit more

PYROTEKNIK committed:
based

PYROTEKNIK committed:
FOV adjustments

PYROTEKNIK committed:
Update menu.lua

too many things i don't know

PYROTEKNIK committed:
Update item.lua

more text color fixing mostly

PYROTEKNIK committed:
Update customizer.lua

lots of shit

PYROTEKNIK committed:
Update imgur_manager.lua

panel doesn't break if you remove the built in text field

PYROTEKNIK committed:
Update stylesheet.lua

new black color,
light mode is slightly tinted towards header color
function for getting the clipped bounds of a panel
shadows draw on stuff

PYROTEKNIK committed:
toggle alternate coloring by right clicking the darkmode button

PYROTEKNIK committed:
Update menu.lua

bug report button

PYROTEKNIK committed:
Merge remote-tracking branch 'upstream/master' into pointshop

PYROTEKNIK committed:
Update imgur_manager.lua

appearance

PYROTEKNIK committed:
Update stylesheet.lua

removed tinting option

PYROTEKNIK committed:
Update customizer.lua

weird error thing

PYROTEKNIK committed:
merge upstream

PYROTEKNIK committed:
Update cl_playerlist.lua

formatted scoreboard icons to be single line

PYROTEKNIK committed:
Update menu.lua

remove click func for removed feature

PYROTEKNIK committed:
Update menu.lua

remove bug reporter button, doesn't really go anywhere useful

PYROTEKNIK committed:
moved imgur manager panel

PYROTEKNIK committed:
Update menu.lua

forgot to adjust income label

swampservers committed:
Merge remote-tracking branch 'origin/master' into pointshop

PYROTEKNIK committed:
oops

PYROTEKNIK committed:
Merge branch 'pointshop' of https://github.com/PYROTEKNIK/contrib into pointshop

PYROTEKNIK committed:
Update stylesheet.lua

whiter shit

PYROTEKNIK committed:
pointshop scroll contents now reach top and bottom bars

PYROTEKNIK committed:
window shadows

PYROTEKNIK committed:
saved textures panel now properly sets text field inside

PYROTEKNIK committed:
Update imgur_manager.lua

none option is redundant

PYROTEKNIK committed:
item names are always white

PYROTEKNIK committed:
Update stylesheet.lua

color adjustments

PYROTEKNIK committed:
Update cl_theaterlist.lua

theater list darker

PYROTEKNIK committed:
Update cl_playerlist.lua

i think not having this was causing errors?

swampservers committed:
Merge remote-tracking branch 'origin/master' into pointshop

swampservers committed:
playerlistchanges

swampservers committed:
Merge branch 'pointshop' of github.com:PYROTEKNIK/contrib into pointshop

swampservers committed:
tweaks, action system

swampservers committed:
fixpadding

swampservers committed:
final

swampservers commented:
Thank you!
